app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        print("Invalid command format.")
+        return
+
+    sourse_file = parts[1]
```

Comment:
There is a typo in the variable name 'sourse_file'. It should be 'source_file' to be descriptive and correct according to the checklist.